packages/composer-json-manipulator/src/ValueObject/ComposerJson.php

```diff
@@ -835,4 +841,32 @@ private function findPosition(string $key, array $items): int | string | bool
     {
         return array_search($key, $items, true);
     }
+
+    /**
+     * @return string[]
+     */
+    public function getProvide(): array
+    {
```

Comment:
Looks like type is `array<string, string>`:
https://github.com/symfony/dependency-injection/blob/855e29cd715ad62bb840c9841fe09a7cde50811f/composer.json#L45-L48

Reply:
Yeah, I mimicked methods for `$replace`. Will fix it. Do you want me to also fix `setReplace` and `getReplace`?

Reply:
Yes, that would be great 👍